### ISSUE TITLE
IP valido não incrementa

### ISSUE THREAD

**Issue by lndgoncalves** [remontti/php-cgnat-netmap-routeros#2]
quando coloca um /30 como ip valido, sempre mantem os mesmo IP inicial para todas as regras.

**Comment by remontti** [comment]
Não utilizo mais esse projeto. 
https://cgnat.remontti.com.br está rodando versão melhorada.

**Comment by lndgoncalves** [comment]
sim, acessei o site e nele que vi o erro, a tabela gera corretamente, mas o
script nao

Em sáb, 14 de jan de 2023 12:08, Rudimar Remontti ***@***.***>
escreveu:
